Google.Ads.GoogleAds/examples/ShoppingAds/AddPerformanceMaxRetailCampaign.cs

```diff
@@ -735,7 +737,10 @@ public string Next()
                 )
             );
 
-            return operations;
+            // The list of operations must be sorted so that all asset
+            // operations come before all the asset group asset operations,
+            // otherwise the API will reject the request.
+            return operations.OrderBy(x => x.AssetOperation is null).ToList();
```

Comment:
This seems a bit hacky. Can we just document and ensure the operations are created in the correct order instead?

Reply:
I agree, I reordered the operations to have them in the correct order.

Note that this will require a change in the docs and, likely, in all other client libraries, since the golden CL, which is Python, sorts the operations afterwards like i did before.

Reply:
I acknowledge, but let's do this in the right manner. Let's submit this one, and file a frontlog entry to fix the logic.